multigeiger/speaker.cpp

```diff
@@ -33,7 +33,7 @@ void IRAM_ATTR isr_audio() {
   // exceptions like "Cache disabled but cached memory region accessed".
   static unsigned int current = 0;  // current period counter
   static unsigned int next = PERIODS(1000);  // periods to next sequencer execution
-  if (current++ < next)
+  if (++current < next)
```

Comment:
if it wants a 1 ms interval, `PERIODS(1000)` will result in `1` meaning that the ISR needs to run every time it is called.

it was run every second time before that fix.